### ISSUE TITLE
Not working on pokernow 0.1.67

### ISSUE THREAD

**Issue by kavimehta** [pj4533/PokerNowGrabber#7]
Not sure if you're still maintaining this, but I had used this a few times with success before the most recent update

**Comment by pj4533** [comment]
Haven't been playing much lately, but if you post more details I might have some time to look at it over the holidays.

**Comment by kavimehta** [comment]
Yeah, sorry for the sparse description. I'm not familiar with Swift so I haven't been able to investigate much myself. Do you use XCode for development? I can try to get things running locally to find more info

As for more details, it was pretty straightforward. I'd used the application 3 times before, following exactly the steps in the ReadMe with no issues — filled in the Hero name, game link, output directory, and NPT cookie. Yesterday, I tried again, but wasn't able to pull anything. The application didn't crash or error, but after filling in the info and hitting "Start Grabbing Hands", nothing was captured as the game was played. I assume this was due to the pokernow update to v0.1.67, but I could be wrong.

**Comment by dhaeb** [comment]
Any progress about this? 
BTW great project!

**Comment by octopoetry** [comment]
Hey! Just discovered this project and I'm hoping that this gets updated to work on the newest version of PokerNow. Would definitely donate if the project stays updated.
